htdocs/info/data/mysql.html

```diff
@@ -82,14 +82,17 @@ <h2>Biomart</h2>
 <h2>Summary of servers and ports</h2>
 
 <p>
-<strong>Important note:</strong> Because we are serving databases on
-both MySQL4 and MySQL5, not all MySQL instances use the default port;
-please ensure that you specify the correct port when trying to
+<strong>Databases prior to release 48</strong> The MySQL version 4 servers providing access to databases from release 47 and earlier have retired. 
```

Comment:
should we not have a column `:` after `Databases prior to release 48`, it feels like its a sub header